### PR TITLE
switch to psutil 1.x syntax

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: templates/configure_slurm.py.j2 [hunk 1]
```diff
@@ -85,8 +85,8 @@ def main():
     template_params = {
         "hostname": gethostname(),
         "user": '{{ galaxy_user_name }}',
-        "cpus": environ.get("SLURM_CPUS", psutil.cpu_count()),
-        "memory": environ.get("SLURM_MEMORY", int(psutil.virtual_memory().total / (1024 * 1024)))
+        "cpus": environ.get("SLURM_CPUS", psutil.NUM_CPUS),
+        "memory": environ.get("SLURM_MEMORY", int(psutil.psutil.TOTAL_PHYMEM / (1024 * 1024)))
```
Comment (jmchilton) [inline templates/configure_slurm.py.j2]: Typo? I gues this should be ``psutil.TOTAL_PHYMEM`` not ``psutil.psutil.TOTAL_PHYMEM``.

Comment (mr-c) [inline templates/configure_slurm.py.j2]: Good catch, thanks!

**Comment by mr-c** [comment]
@jmchilton ready for review and merge

**Comment by mr-c** [comment]
Thanks!

**Comment by jmchilton** [comment]
Thanks for the contribution @mr-c!

**Pull request merged**